custom/enikshay/tests/test_adherence_updates.py

```diff
@@ -144,15 +143,15 @@ def _create_episode_case(self, adherence_schedule_date_start, adherence_schedule
     def _create_adherence_cases(self, adherence_cases):
         self.factory.create_or_update_cases([
             get_adherence_case_structure(
-                adherence_date.strftime('%Y-%m-%d-%H-%M'),
+                "adherence{}".format(i),
```

Comment:
why do you do this? this will name the adherence cases `adherence0`, `adherence1` etc, right? 
Wouldn't that lead to duplicate names each time this task is run?